Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN set -x \
   # Install cypress dependencies & JRE (required by Jenkins)
   && echo 'deb http://ftp.debian.org/debian jessie-backports main' >> /etc/apt/sources.list \
   && apt-get update \
-  && apt-get install -y xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 \
+  && apt-get install -y xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 lsof \
```

Comment:
just a nitpick, but where do we use the lsof? I searched this page and did not found another use.

Reply:
it is used in the `run-integration-tests` script which was already merged - we forgot to add lsof :(